app/containers/MessageBox/index.js

```diff
@@ -818,63 +819,95 @@ class MessageBox extends Component {
 			returnKeyType: 'send'
 		} : {};
 
+		const recordAudio = showSend || !Message_AudioRecorderEnabled ? null : (
```

Comment:
I can't understand the main purpose of these changes, since, you can re-order the buttons inside the `MessageBox/buttons`. Why did you bring these logic here?

Reply:
Related to https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/2195/files/d2f5cffb24342c86009fc8513690d16f3ea123a0#r442894161, resolving